zerver/lib/integrations.py

```diff
@@ -1023,14 +1031,38 @@ def __init__(self, name: str, *args: Any, **kwargs: Any) -> None:
 
 # Each of these integrations have their own Zulip repository in GitHub.
 STANDALONE_REPO_INTEGRATIONS: list[Integration] = [
-    Integration("errbot", ["meta-integration", "bots"]),
+    Integration(
+        "errbot",
+        ["meta-integration", "bots"],
+        [
+            FixturelessScreenshotConfigOptions(
+                thread_messages=[
```

Comment:
It might need some work, but we'd want all this hardcoded string data to be in fixtureless_integrations.py.

And it looks odd to just repeat things twice - the entire purpose is to demonstrate the bot.